components/Payment/Form.js

```diff
@@ -195,10 +195,10 @@ class PaymentForm extends Component {
           amount: total,
           usage: 'reusable',
           card: {
-            number: values.cardNumber,
-            cvc: values.cardCVC,
-            exp_month: values.cardMonth,
-            exp_year: values.cardYear
+            number: values.cardNumber.trim(),
+            cvc: values.cardCVC.trim(),
+            exp_month: parseInt(values.cardMonth),
```

Comment:
`parseInt('08') === 0` in IE11 – see https://nathanhoad.net/javascript-parseint-problem/

Use `+values.cardMonth` instead:
```suggestion
            exp_month: +values.cardMonth,
```